asn1/CommonLedgerProtocol.asn

```diff
@@ -21,81 +21,53 @@ SideProtocolData ::= SEQUENCE OF SEQUENCE {
   protocolData OCTET STRING
 }
 
-PacketRequest ::= SEQUENCE {
-  packet InterledgerPacket,
+Ack ::= SEQUENCE {
+  ack NULL,
```

Comment:
ok, but then maybe we could just name them:
`Request`
`Response`
`RequestWithPacket`
`ResponseWithPacket`
`RequestWithSideData`
`ResponseWithSideData`
`RequestWithPacketAndSideData`
`ResponseWithPacketAndSideData`